src/stacks/nestedSlackNotifications.ts

```diff
@@ -0,0 +1,56 @@
+import * as cdk from '@aws-cdk/core';
+import * as cfn from '@aws-cdk/aws-cloudformation';
+import * as lambda from '@aws-cdk/aws-lambda';
+import * as nodejsLambda from '@aws-cdk/aws-lambda-nodejs';
+import { NestedSNSStack } from './nestedSns';
+import * as config from '../utils/config';
+
+export class NestedSlackNotificationsStack extends cfn.NestedStack {
```

Comment:
I would think it would be better to have this in the same stack as sns topics. Don't really see reason to have separate stack for just one lambda